server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_5__add_flow_instance_index.sql

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+alter table `flow_instance` add index `flow_instance_parent_ins_id`(`parent_instance_id`);
+
+alter table `flow_instance_node_task` add index `flow_instance_node_task_flow_instance_id`(`flow_instance_id`);
+
+alter table `flow_instance` add index `flow_instance_parent_oid_ct_id`(`organization_id`,`create_time`,`id`);
+
+CREATE VIEW `list_flow_instance_view` AS
+select
+    `odc_metadb3`.`flow_instance`.`id` AS `id`,
+    `odc_metadb3`.`flow_instance`.`create_time` AS `create_time`,
```

Comment:
@PeachThinking What is the relationship between this view and your previous view?

Reply:
replace previous view.